object-construction-checker/src/main/java/org/checkerframework/checker/objectconstruction/ObjectConstructionChecker.java

```diff
@@ -20,7 +20,13 @@
   ObjectConstructionChecker.COUNT_FRAMEWORK_BUILD_CALLS,
   ObjectConstructionChecker.DISABLED_FRAMEWORK_SUPPORTS,
 })
-@StubFiles({"Socket.astub", "NotOwning.astub", "Stream.astub", "AlwaysCallEmpty.astub", "IOUtils.astub"})
+@StubFiles({
```

Comment:
This was an extraneous re-formatting done by Spotless

Reply:
The `always-call-checker` branch is failing spotless right now; my branch with the MustCall Checker required the same change.